Task_HelloWorld/index.css

```diff
@@ -1 +1,7 @@
 /* Write your code below for HELLO WORLD TASK*/
+<h>Hello World:</h>
+<p>In HTML, you can use various tags to format your content.
+    For example, you can use the <b>Bold</b> tag to make text bold,
+    the <i>Italic</i> tag to italicize text, and the <u>Underline</u> tag to underline text. 
```

Comment:
The closing tag for the 'br' tag is missing. It should be '</br>' instead of '<br>'.